L/libemos/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+using BinaryBuilder
+
+name = "libemos"
+version = v"4.5.9"
+sources = [
+    ArchiveSource("https://confluence.ecmwf.int/download/attachments/3473472/libemos-$version-Source.tar.gz", "e57e02c636dc8f5ccb862a103789b1e927bc985b1e0f2b05abf4f64e86d2f67f"),
+]
+
+script = raw"""
+cd libemos-*-Source
+mkdir build ; cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DENABLE_ECCODES=ON \
+    -DGRIB_API_LIBRARIES=${libdir}/libeccodes.so \
+    -DGRIB_API_INCLUDE_DIRS=${prefix}/include \
+    ..
+make -j${nproc}
+make install
+install_license ../LICENSE
+"""
+
+platforms = supported_platforms()
+
+products = [
+    # LibraryProduct("libemos", :emos),
+    FileProduct("lib/libemos.a", :emos), # libemos is not designed to be build as a shared library (see libemos-$version-Source/CMakeLists.txt:39)
```

Comment:
Hi, thanks for your contribution!  If this can't provide a shared library, how are you planning to use this from Julia?  _If_ you're planning to use this library from Julia :slightly_smiling_face:

Reply:
Hi! I don't plan to use it from Julia, it's just a [dependency for flex_extract](https://www.flexpart.eu/flex_extract/Installation/local.html) for which I intend to build binaries for Julia (hoping that it will work, I'm kind of new to this :slightly_smiling_face:)